docs/animations/ImplicitAnimations.md

```diff
@@ -1,17 +1,19 @@
 ---
 title: Implicit Animations XAML Attached Properties
-ms.date: 11/22/2017
+author: nmetulev
 description: The Implicit Animations Attached Properties enable implicit animations to be defined in your XAML code
 keywords: windows 10, uwp, uwp community toolkit, uwp toolkit, composition animations, animation, implicit animations, XAML, implicit, composition, show animation, hide animation
 ---
 
 # Implicit Composition Animations in XAML
 
-Implicit Animations are Composition Animations that are used to describe how and when animations occur as a response to direct property changes, such as Opacity or Offset. Show and Hide animations describe the animation to be applied to an element when the Visibility is changed, or the element is added/removed to the visual tree
+Implicit Animations are Composition Animations that are used to describe how and when animations occur as a response to direct property changes, such as Opacity or Offset. Show and Hide animations describe the animation to be applied to an element when the Visibility is changed, or the element is added/removed to the visual tree.
 
-The Implicit Animations Attached Properties enable implicit animations to be defined in your XAML code by using the [Composition Animation](CompositionAnimations.md) XAML objects. This allows animations to be defined directly on the element, or defined as XAML resources and applied to any XAML element. 
+**Prerequisites** - [Composition Animation in XAML](https://docs.microsoft.com/en-us/windows/uwpcommunitytoolkit/animations/compositionanimations)
 
-The Implicit Animations Attached Properties can be used in combination with the VisualExtensions. This works well when used in Storyboards.
+The Implicit Animations Attached Properties enable implicit animations to be defined in your XAML code by using the [Composition Animation](https://docs.microsoft.com/en-us/windows/uwpcommunitytoolkit/animations/compositionanimations) XAML objects. This allows animations to be defined directly on the element, or defined as XAML resources and applied to any XAML element. 
+
+The Implicit Animations Attached Properties can be used in combination with the [VisualExtensions](https://docs.microsoft.com/en-us/windows/uwpcommunitytoolkit/extensions/viewextensions). This works well when used in Storyboards.
```

Comment:
VisualExtensions should be ViewExtensions

Reply:
haha Actually, ViewExtensions should be VisualExtensions. Wrong link 😄